javascript/auth-code-client-side/README.md

```diff
@@ -0,0 +1,51 @@
+# Authorization Code Sample
+
+This is a fully client-side implementation of the OAuth 2.0
+Authorization Code Grant. It's meant to be a reference implementation of
+using the Getty API with users using their own Getty or iStock
+credentials.
+
+This form of Authorization Code does not use a client secret. It does
+make use of the [PKCE](https://datatracker.ietf.org/doc/html/rfc7636)
+extension to aid in keeping the access token secure from any malicious
+scripts or apps on the user's device.
+
+This project does not rely on _any_ third-party libraries or packages.
+
+## Running locally
+
+After cloning this repository, the simplest way to test it out is by
+using [Docker](https://docs.docker.com/get-docker/).
+
+```sh
+git clone https://github.com/gettyimages/gettyimages-api.git
+cd javascript/auth-code-client-side
```

Comment:
If I type these commands, I need to `cd` into the working directory itself too:  ` cd gettyimages-api/javascript/auth-code-client-side`

Reply:
Yep, I missed that.